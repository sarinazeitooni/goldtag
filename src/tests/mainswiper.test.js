import React from 'react';
import {shallow} from 'enzyme';
import MainSwiper from "../components/swiper/mainswipper";
describe("main swiper component" , ()=>{
    it("should match snap shot" , ()=>{
        const wrapper = shallow(<MainSwiper/>);
        expect(wrapper).toMatchSnapshot();
    });
   it("should render correctly" , ()=>{
       const wrapper = shallow(<MainSwiper/>);
       const items = wrapper.find('div.swiper-item');
       expect(items.exists()).toBe(true);
    });
});