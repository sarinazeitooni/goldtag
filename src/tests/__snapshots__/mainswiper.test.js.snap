// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main swiper component should match snap shot 1`] = `ShallowWrapper {}`;
